joomla/joomla-cms
# Revised field_group with new base

Pull Request for Issue #31385 and related to #31396. Revised version of #31408

Summary of Changes

Added code so that field groups can have alternatives to ul layouts and headings.

Testing Instructions

Apply the patch

Create two field groups:
Ingredients
Set Options / Render Tag to ul
Procedure
Set Options / Render Tag to ol

Create fields with Ingredients as Field Group
with Options / Render Options / Label set to Hide (Optional)
Item 1
Item 2
Item 3

Create fields with Procedure as Field Group
with Options / Render Options / Label set to Hide (Optional)
Stage 1
Stage 2
Stage 3

Edit an article
Fill out the Ingredients form Tab
Water
Tea
Milk
Fill out the Procedure form Tab
Boil the water
Pour the water on the tea
Add the milk

Actual result BEFORE applying this Pull Request

A single unordered list

Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/1560390/145216320-96a2e349-5183-451e-8649-4e3f9f58b214.png)

Documentation Changes Required

Yes - I will do that when merged.
